### PR TITLE
add keymap with split spacebar, ansi, hhkb features

### PR DESCRIPTION
**Pull request by calbatr0ss**
## Description

Added the `calbatr0ss` layout that has the following features:
- split spacebar
- ansi enter key
- hhkb delete key and Fn key placement
- a mac layer
- a media key layer

I have documented the layers and their keymaps in the corresponding `calbatr0ss/keymap.c` file, as well as the `info.json` for the DZ60. I have tested this firmware on my personal keyboard.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -44,6 +44,7 @@ doxygen/
 .project
 .settings/
 .idea
+.vscode
```
Comment (fauxpark) [inline .gitignore]: Please remove this line, there are already files checked into this directory.

Comment (calbatr0ss) [reply .gitignore]: removed

**Comment by calbatr0ss** [comment]
I pulled from master before making this PR, it looks like the Travis CI is having trouble making keymaps other than mine that I didn't edit. Is there a protocol I should follow?

**Comment by fauxpark** [comment]
That should be fine - you haven't made any changes to the keyboard-level files that would introduce errors like that, so they are pre-existing issues with those keymaps.

**Comment by drashna** [comment]
Exactly.  Travis CI errors are unrelated to your changes, so they can be ignored. 

Thanks!

**Pull request merged**